pkg/glusterfs/utils/config.go

```diff
@@ -4,23 +4,8 @@ package utils
 const (
 	KB int64 = 1000
 	MB int64 = 1000 * KB
-	GB int64 = 1000 * MB
-	TB int64 = 1000 * GB
 )
 
-// RoundUpSize calculates how many allocation units are needed to accommodate
-// a volume of given size.
-// RoundUpSize(1500 * 1000*1000, 1000*1000*1000) returns '2'
-// (2 GB is the smallest allocatable volume that can hold 1500MiB)
-func RoundUpSize(volumeSizeBytes int64, allocationUnitBytes int64) int64 {
-	return (volumeSizeBytes + allocationUnitBytes - 1) / allocationUnitBytes
-}
-
-// RoundUpToGB rounds up given quantity upto chunks of GB
-func RoundUpToGB(sizeBytes int64) int64 {
-	return RoundUpSize(sizeBytes, GB)
-}
-
```

Comment:
@Madhu-1 dont remove the util functions. Let it be there. Rest looks good to me.